java/src/jmri/configurexml/StoreXmlUserAction.java

```diff
@@ -36,8 +33,8 @@ public StoreXmlUserAction(String s) {
     public void actionPerformed(ActionEvent e) {
         JFileChooser userFileChooser = getUserFileChooser();
         userFileChooser.setDialogType(javax.swing.JFileChooser.SAVE_DIALOG);
-        userFileChooser.setApproveButtonText(Bundle.getMessage("ButtonSave")); // is in jmri.NBBundle
-        userFileChooser.setDialogTitle(rb.getString("StorePanelTitle"));
+        userFileChooser.setApproveButtonText(Bundle.getMessage("ButtonSave"));  // NOI18N
+        userFileChooser.setDialogTitle(Bundle.getMessage("StoreTitle"));  // NOI18N
```

Comment:
No need to change this. But I think that `// NOI18N` is not needed when the string is in `Bundle.getMessage()`.

Reply:
According to https://www.jmri.org/help/en/html/doc/Technical/I8N.shtml, bundle keys are annotated.

```
          Sensor thisSensorVariableNameIsInEnglish;

           String message = "THAT_ONE_MESSAGE";  // NOI18N
           JLabel jl1 = new JLabel(Bundle.getMessage(message));

           JLabel jl2 = new JLabel(Bundle.getMessage("LABELKEY"));  // NOI18N

           log.debug("The process failed account of user error"); // NOI18N

           // This comment is in English and need not be annotated w.r.t. internationalization
```

Reply:
Thank you. I had missed that.